modules/sharedaddy/sharing-service.php

```diff
@@ -748,8 +748,17 @@ function sharing_display( $text = '', $echo = false ) {
 		$show = false;
 	}
 
-	// Allow to be used on P2 ajax requests for latest posts.
-	if ( defined( 'DOING_AJAX' ) && DOING_AJAX && isset( $_REQUEST['action'] ) && 'get_latest_posts' == $_REQUEST['action'] ) {
+	/**
+	 * Filter the default ajax action name
+	 *
+	 * @module sharedaddy
+	 *
+	 * @since 5.1.1
+	 */
```

Comment:
Here you are missing the parameter that can be modified with the filter:

```suggestion
	 * @param string $sharing_ajax_action_name Name of the Sharing buttons' Ajax action.
	 */
```